source/core/ut_suite_cache_manager.pkb

```diff
@@ -321,30 +273,29 @@ create or replace package body ut_suite_cache_manager is
   end;
   
   function get_cached_suite_rows(
-    a_schema_paths     ut_path_items,
-    a_random_seed      positive := null,
-    a_tags             ut_varchar2_rows := null
+    a_suites_filtered ut_suite_cache_rows 
   ) return ut_suite_cache_rows is
     l_results         ut_suite_cache_rows := ut_suite_cache_rows();
-    l_suite_items     ut_suite_cache_rows := ut_suite_cache_rows();
-    l_schema_paths    ut_path_items;
-    l_tags            ut_varchar2_rows := coalesce(a_tags,ut_varchar2_rows());
   begin     
 
-    l_schema_paths := a_schema_paths;
-    l_suite_items := get_suite_items(a_schema_paths);
-    if l_tags.count > 0 then
-      l_suite_items := get_tags_suites(l_suite_items,l_tags);
-    end if;
-    
-    open c_get_bulk_cache_suite(l_suite_items);
+    open c_get_bulk_cache_suite(a_suites_filtered);
     fetch c_get_bulk_cache_suite bulk collect into l_results;
     close c_get_bulk_cache_suite;
       
     return l_results;
   end;
     
-  
+  function get_cached_suites(
```

Comment:
This function is not needed anymore. It is an empty shell for call:
```
    l_suite_items := get_suite_items(a_schema_paths);
    return l_suite_items;
```
That can be reduced to: `  return get_suite_items(a_schema_paths);`

And so we can simply use function `get_suite_items` directly instead and remove this shell function.